src/PIL/Image.py

```diff
@@ -216,6 +217,15 @@ class Quantize(IntEnum):
     RLE = core.RLE
     FIXED = core.FIXED
 
+Tile = NamedTuple(
+    "Tile",
+    [
+        ("encoder_name", str),
+        ("extents", Tuple[int, int, int, int]),
+        ("offset", int),
+        ("tile_args", Tuple),
+    ],
+)
```

Comment:
Could you say a few words about why this is helpful?

Reply:
I've added it to understand what tile structure is. I didn't found any doc for it, so i scouted sources to understand what everything mean